src/BPEssentials/FileChecker.cs

```diff
@@ -7,6 +7,13 @@ namespace BPEssentials
 {
     public static class FileChecker
     {
+        private static readonly Dictionary<string, string> RequiredDirectorys = new Dictionary<string, string>
```

Comment:
Rename to `RequiredDirectories`.

```suggestion
        public static Dictionary<string, string> RequiredDirectories { get; }= new Dictionary<string, string>
```